website/docs/upgrades/1.0_to_1.1/automatic_schema_matching.md

```diff
@@ -92,3 +98,78 @@ cs.store(group="db",
 </div>
 </div>
 
+</details>
+
+### Rename the config file
+Steps:
+1. Rename the config file. Common choices are `custom_` or `my_` prefix, e.g. `custom_mysql.yaml`.
+2. Add the schema to the Defaults List of the extending config file.
```

Comment:
This makes it look like things will work as before after performing these 2 steps. But that may not be the case => I think it would be important to add a "step 3" along the lines of:
```
3. Update references to the old file name accordingly, e.g. on the command-line `db=mysql`
   would become `db=custom_mysql`, and in a defaults list `db: mysql` would become
   `db: custom_mysql` (see below for an example)
```
Then in the example below, replace the config store code block (which doesn't change anyway) with a config file that has `db: mysql` in its defaults list.

Reply:
Good catch.